src/snapred/backend/recipe/algorithm/CustomGroupWorkspace.py

```diff
@@ -1,8 +1,14 @@
 import json
 
-from mantid.api import AlgorithmFactory, PythonAlgorithm
+from mantid.api import AlgorithmFactory, MatrixWorkspaceProperty, PropertyMode, PythonAlgorithm
 from mantid.kernel import Direction
-from mantid.simpleapi import CreateWorkspace, GroupWorkspaces, LoadDetectorsGroupingFile, LoadInstrument
+from mantid.simpleapi import (
```

Comment:
Is there a reason why we have to import from `simpleapi` here and not use `MantidSnapper`?

Reply:
I'm suspicious that this is older than `MantidSnapper`. I'll update the imports.

Reply:
Pete is correct